sumologic.sh

```diff
@@ -0,0 +1,210 @@
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "OS/Linux/System",
+      "pathExpression": "/var/log/mail.log"
+    },
+    {
+      "name": "Secure",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "OS/Linux/Security",
+      "pathExpression": "/var/log/secure"
+    },
+    {
+      "name": "Syslog File",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "OS/Linux/System",
+      "pathExpression": "/var/log/syslog"
+    },
+    {
+      "name": "chef_erchef",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "chef/erchef",
+      "pathExpression": "/var/log/opscode/opscode-erchef/current"
+    },
+    {
+      "name": "chef_nginx",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "chef/nginx",
+      "pathExpression": "/var/log/opscode/nginx/*.log"
+    },
+    {
+      "name": "chef_bookshelf",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "chef/bookshelf",
+      "pathExpression": "/var/log/opscode/bookshelf/current"
+    },
+    {
+      "name": "chef_redis_lb",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "chef/redis_lb",
+      "pathExpression": "/var/log/opscode/redis_lb/current"
+    },
+    {
+      "name": "chef_oc_id",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "chef/oc_id",
+      "pathExpression": "/var/log/opscode/oc_id/current"
+    },
+    {
+      "name": "chef_oc_bifrost",
+      "sourceType": "LocalFile",
+      "automaticDateParsing": true,
+      "multilineProcessingEnabled": false,
+      "useAutolineMatching": true,
+      "forceTimeZone": false,
+      "timeZone": "UTC",
+      "category": "chef/oc_bifrost",
+      "pathExpression": "/var/log/opscode/oc_bifrost/current"
+    },
+    {
+      "name": "chef_manage_redis",
```

Comment:
May no longer need this file if redis is gone.  Will want to log on to a server and check the /var/log/opscode directory for the files that are there. (that is how I came up with the initial list.